backend/tests/integration/indexer/test_cli.py

```diff
@@ -192,3 +192,82 @@ def test_remap_index_when_there_are_more_than_one_indexes_with_same_name_but_dif
 
     list(current_search.delete("*"))
     current_search._current_suffix = None
+
+
+def test_cli_create_aliases(inspire_app, cli):
+    prefix = "test-cli-create-aliases-prefix-"
+    with override_config(SEARCH_INDEX_PREFIX=prefix):
+        list(current_search.create(ignore_existing=True))
+        result = cli.invoke(["index", "create-aliases", "--yes-i-know"])
+    assert result.exit_code == 0
+    assert (
+        result.output
+        != "This command can be executed only if SEARCH_INDEX_PREFIX is set.\n"
+    )
+    assert "does not contain current prefix" in result.output
+
+    aliases = []
+    for x in current_search.client.indices.get_alias().values():
+        aliases.extend(x["aliases"].keys())
+
+    assert "records-hep" in aliases
+    assert "records-authors" in aliases
+    assert "records-seminars" in aliases
+    assert "records-jobs" in aliases
+    assert "records-institutions" in aliases
+    assert "records-journals" in aliases
+    assert "records-data" in aliases
+    assert "records-experiments" in aliases
+    assert "records-conferences" in aliases
```

Comment:
could we also test where they are pointing?

Reply:
:+1: